PetaPoco.Tests.Integration/Databases/OracleTests/OracleQueryTests.cs

```diff
@@ -54,11 +156,22 @@ public override void QueryMultiple_ForSingleResultsSetWithMultiPoco_ShouldReturn
             var pdName = pd.Columns.Values.First(c => c.PropertyInfo.Name == "Name").ColumnName;
             var odPersonId = od.Columns.Values.First(c => c.PropertyInfo.Name == "PersonId").ColumnName;
 
-            var sql = $@"SELECT * FROM {DB.Provider.EscapeTableName(od.TableInfo.TableName)} o
-                         INNER JOIN {DB.Provider.EscapeTableName(pd.TableInfo.TableName)} p ON p.{DB.Provider.EscapeSqlIdentifier(pdId)} = o.{DB.Provider.EscapeSqlIdentifier(odPersonId)}
-                         WHERE p.{DB.Provider.EscapeSqlIdentifier(pdName)} = @0
-                         ORDER BY 1 DESC
-                         LIMIT 1;";
+            //Oracle 12c and above only
+    //        var sql = $@"SELECT *
+    //FROM {DB.Provider.EscapeTableName(od.TableInfo.TableName)} o
+    //    INNER JOIN {DB.Provider.EscapeTableName(pd.TableInfo.TableName)} p ON p.{DB.Provider.EscapeSqlIdentifier(pdId)} = o.{DB.Provider.EscapeSqlIdentifier(odPersonId)}
+    //WHERE p.{DB.Provider.EscapeSqlIdentifier(pdName)} = @0
+    //ORDER BY 1 DESC
+    //FETCH FIRST 1 ROWS ONLY";
+
+            var sql = $@"SELECT *
+    FROM (SELECT *
+        FROM {DB.Provider.EscapeTableName(od.TableInfo.TableName)} o
+            INNER JOIN {DB.Provider.EscapeTableName(pd.TableInfo.TableName)} p
+                ON p.{DB.Provider.EscapeSqlIdentifier(pdId)} = o.{DB.Provider.EscapeSqlIdentifier(odPersonId)}
+        WHERE p.{DB.Provider.EscapeSqlIdentifier(pdName)} = @0
+        ORDER BY 1 DESC)
+    WHERE ROWNUM <= 1";
 
```

Comment:
Same as above comment re indentation.

Reply:
Those lines should've actually been against the left gutter. So when I inspected `sql`, I saw
```sql
SELECT *
						FROM table1
							INNER JOIN table2
								ON table1.id = table2.id
						WHERE table1.column1 = 'value'
```
, but expected
```sql
SELECT *
FROM table1
	INNER JOIN table2
		ON table1.id = table2.id
WHERE table1.column1 = 'value'
```
So my "OCD" kicked in and I removed all the excess tabs in order to give me a statement that's already formatted like the ones you see in the Build and Setup scripts. Hope it's ok?